### ISSUE TITLE
Blur not working properly

### ISSUE THREAD

**Issue by LucasFebatis** [Akylas/nativescript-material-components#21]
The blur event only works when a keyboard execute next button, but when the focus of the component is removed by clicking on another component the blur event is not triggered.

In Nativescript TextField this runs in both cases.

Look:

code:
html:
```
<TextField hint="Enter date" (focus)="onFocus($event)" (blur)="onBlur($event)" ></TextField>

<MDTextField #tvNumCard (focus)="onFocus($event)" (blur)="onBlur($event)" class="input" variant="filled" hint="Field 1" returnKeyType="next"></MDTextField>
<MDTextField class="input" variant="filled" hint="Field 2" returnKeyType="next"></MDTextField>
<MDTextField class="input" variant="filled" hint="Field 3" returnKeyType="next"></MDTextField>
<MDTextField class="input" variant="filled" hint="Field 4" keyboardType="datetime" returnKeyType="done"></MDTextField>
```

ts
```
onFocus(event) {
    console.log("onFocus")
}
    
onBlur(event) { 
    console.log("onBlur")
}
```

images:

<img width="311" alt="Captura de Tela 2019-06-09 às 23 51 30" src="https://user-images.githubusercontent.com/28310725/59169690-a9252700-8b11-11e9-9883-3141a3d446dd.png">

- Cursor on Enter Date, onFocus of Enter Date is fired

<img width="312" alt="Captura de Tela 2019-06-09 às 23 51 43" src="https://user-images.githubusercontent.com/28310725/59169727-cc4fd680-8b11-11e9-98ce-4d94323ce866.png">

- Then it is clicked on Field 1, and the onBlur of the Enter Date is fired and the onFocus of Field 1 is also fired.

<img width="307" alt="Captura de Tela 2019-06-09 às 23 51 55" src="https://user-images.githubusercontent.com/28310725/59169768-f6a19400-8b11-11e9-8a39-788f936488b5.png">

- Finally Field 2 is clicked, and the Field 1 onBlur that is programmed to be done does not happen.

<img width="312" alt="Captura de Tela 2019-06-09 às 23 52 18" src="https://user-images.githubusercontent.com/28310725/59169796-0faa4500-8b12-11e9-8f71-a5464d6e1447.png">

- But if I click on Field 1 and click next on the keyboard, Field 1 onBlur is triggered

Nativescript: 5.4.1
Angular: 8.0.0

**Comment by farfromrefug** [comment]
@LucasFebatis thanks i managed to reproduce it. Will fix it soon

**Comment by farfromrefug** [comment]
@LucasFebatis should be fixed in 2.2.16

**Comment by LucasFebatis** [comment]
@farfromrefug Thank a lot, I'll close the issue
